diffkemp/simpll/DifferentialFunctionComparator.cpp

```diff
@@ -401,6 +401,38 @@ int DifferentialFunctionComparator::cmpValues(const Value *L,
     return result;
 }
 
+/// Specific comparing of constants. If one of them (or both) is a cast
+/// constant expression, compare its interal value.
```

Comment:
Should be "integral value"?

Reply:
It's actually a different typo - I meant "internal", nevertheless it's better to call it "operand".